### PR TITLE
what in ternation

### PR DESCRIPTION
**Pull request by VishramKidPG123**
adding a bunch of ternary operators 
this will fix the "i love dpy" issue
hopefully it doesn't break stuff but it should just keep things exactly the same but with less code

### CODE REVIEW

**Comment by milena-kos** [comment]
the problem is dpy doesnt support setting that value to none

**Comment by VishramKidPG123** [comment]
A callable may work though I've never used them

**Comment by ZakChrom** [comment]
Why is there a merge commit in the commits? Just rebase the pull

**Comment by milena-kos** [comment]
> the problem is dpy doesnt support setting that value to none

which btw is a horrible idea i see no good reason they would do this

**Comment by VishramKidPG123** [comment]
I think I found a solution (yay stackoverflow)
I'll add it to my pr in a bit
https://stackoverflow.com/a/56394233

**Comment by ZakChrom** [comment]
:staring_cat: why no rebase

File: main.py [hunk 1]
```diff
@@ -1028,41 +1010,15 @@ async def dark_market_cutscene(interaction):
...
-                                                allowed_mentions=discord.AllowedMentions.none())
-                        else:
-                            await send_target.send(coughstring.replace("{username}", message.author.name.replace("_", "\\_"))
+                    await send_target.send(coughstring.replace("{username}", message.author.name.replace("_", "\\_"))
                                                             .replace("{emoji}", str(icon))
                                                             .replace("{type}", le_emoji)
                                                             .replace("{count}", str(add_cat(message.guild.id, message.author.id, le_emoji, silly_amount)))
                                                             .replace("{time}", caught_time[:-1]) + suffix_string,
+                                                *[view=view] if view else [],
+                                                *[thread=discord.Object(message.channel.id)] if thread_id else [],
```
Comment (milena-kos) [inline main.py:1021]: im deeply horrified we might summon satan at some point this is getting unholy

Comment (VishramKidPG123) [reply main.py:1021]: whoopsie

**Comment by VishramKidPG123** [comment]
okay this is as much ternary-operator-ing and match-case-ing that I am willing to do
please someone go through this and make sure I'm not breaking anything :staring_cat:

**Comment by milenakos** [comment]
this is conflicting cus of db changes btw

**Comment by milenakos** [comment]
im just gonna cherry pick out of this and then close

**Pull request closed without merging**